testsuite/features/support/commonlib.rb

```diff
@@ -37,7 +37,7 @@ def compute_channels_to_leave_running
     os_family = node.os_family
     next unless ['sles', 'rocky'].include?(os_family)
     os_version = os_version.split('.')[0] if os_family == 'rocky'
-    raise "Can't build list of reposyncs to leave running" unless %w[12-SP4 12-SP5 15-SP3 15-SP4 8].include? os_version
+    log "Can't build list of reposyncs to leave running" unless %w[15-SP3 15-SP5 8].include? os_version
```

Comment:
```suggestion
    log "Can't build list of reposyncs to leave running" unless %w[15-SP3 15-SP4 8].include? os_version
```

Reply:
I should have left a comment in the code, this is temporary to trigger the `log` and will be reverted asap